src/java.desktop/share/legal/lcms.md

```diff
@@ -44,12 +42,12 @@ SoftSurfer makes no warranty for this code, and cannot be held
 liable for any real or imagined damage resulting from its use.
 Users of this code must verify correctness for their application.
 
-
-</pre>
+<pre>
```

Comment:
Update looks good to me. Shouldn't this stay as "/pre" though to close the opening tag on line 4?

Reply:
Just fixed